forta-emitting-upgraded/src/agent.js

```diff
@@ -33,6 +33,9 @@ function provideHandleTransaction(getContracts) {
     );
 
     txFiltered.forEach((tx) => {
+      const { from } = tx;
+      const { implementation } = tx.args;
+      const fromLowerCase = from.toLowerCase();
```

Comment:
I got "Cannot read property 'toLowerCase' of undefined" when I tried the test transaction from the README. I think the problem is that the event doesn't have "from" property. Maybe you meant "address"?